src/routes/_utils/ajax.js

```diff
@@ -51,7 +51,7 @@ async function _fetch (url, fetchOptions, options) {
 async function _putOrPostOrPatch (method, url, body, headers, options) {
   const fetchOptions = makeFetchOptions(method, headers, options)
   if (body) {
-    if (body instanceof FormData) {
+    if (body instanceof FormData || body instanceof URLSearchParams) {
       fetchOptions.body = body
```

Comment:
It seems that, by default, since the `Content-Type` is not set here, the browser (Chromium at least) sets

- `multipart/form-data; boundary=----WebKitFormBoundaryCBpNkZBPd2t28viK` for `FormData`
-  `application/x-www-form-urlencoded; charset=utf-8` for `URLSearchParams`

This is really surprising behavior to me. To me it would make more sense to explicitly specify the two content-types in the two different cases.

Reply:
That seems to be documented in https://fetch.spec.whatwg.org/#bodyinit-unions

Reply:
Fascinating, thanks!